src/hooks/useFetch.js

```diff
@@ -21,7 +21,7 @@ const ERROR_CODES = {
 const getAuthorizationHeader = () => {
   const username = UIDatabase.getSetting(SYNC_SITE_NAME);
   const password = UIDatabase.getSetting(SYNC_SITE_PASSWORD_HASH);
-  return { authorization: getAuthHeader(username, password) };
+  return getAuthHeader(username, password);
```

Comment:
this doesn't look like bugsnagging... 🤔 
I thought it might be cross-contamination or offroad, but I see now after looking below. Go for it!